workflows/proteomics/metaquantome-function/metaquantome_function_wf.ga

```diff
@@ -0,0 +1,636 @@
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [],
+            "label": null,
+            "name": "metaQuantome: database",
+            "outputs": [
+                {
+                    "name": "outfile",
+                    "type": "tar"
+                }
+            ],
+            "position": {
+                "left": 0,
+                "top": 164.1260045776367
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_db/metaquantome_db/2.0.2+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "e63db1aa82ee",
+                "name": "metaquantome_db",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"__input_ext\": \"data\", \"chromInfo\": \"/opt/galaxy/tool-data/shared/ucsc/chrom/?.len\", \"databases\": [\"ncbi\", \"go\", \"ec\"], \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "2.0.2+galaxy0",
+            "type": "tool",
+            "uuid": "1ae509c1-52ea-4fef-ad4f-bcc3b836ed08",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "metaQuantome databases archive File",
+                    "output_name": "outfile",
+                    "uuid": "2c90e06b-9d2a-40c5-bedd-c50c88c47d01"
+                }
+            ]
+        },
+        "1": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_sample/metaquantome_sample/2.0.2+galaxy0",
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [],
+            "label": null,
+            "name": "metaQuantome: create samples file",
+            "outputs": [
+                {
+                    "name": "samples_file",
+                    "type": "tabular"
+                }
+            ],
+            "position": {
+                "left": 6.015625,
+                "top": 310.1728795776367
+            },
+            "post_job_actions": {},
+            "tool_id": "toolshed.g2.bx.psu.edu/repos/galaxyp/metaquantome_sample/metaquantome_sample/2.0.2+galaxy0",
+            "tool_shed_repository": {
+                "changeset_revision": "06f6826d380c",
+                "name": "metaquantome_sample",
+                "owner": "galaxyp",
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            },
+            "tool_state": "{\"samps_args\": {\"samps_src\": \"build\", \"__current_case__\": 0, \"samps\": [{\"__index__\": 0, \"group_name\": \"T2\", \"col_names\": \"T2_A1,T2_B1\"}, {\"__index__\": 1, \"group_name\": \"T4\", \"col_names\": \"T4A_1,T4B_1\"}, {\"__index__\": 2, \"group_name\": \"T7\", \"col_names\": \"T7A_1,T7B_1\"}]}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
```

Comment:
This step is strange. Why tie a general WF to specific sample and group names? All of these should probably be WF input params.